webapp/app/js/perf.js

```diff
@@ -28,3 +41,207 @@ perf.factory('getSeriesSummary', [ function() {
              options: options, subtestSignatures: subtestSignatures };
   };
 }]);
+
+
+perf.factory('getCounterMap', [ 'isReverseTest',
+  function(isReverseTest) {
+    return function(testName, originalData, newData) {
+      var cmap = {'originalGeoMean': NaN, 'originalRuns': 0, 'originalStddev': NaN,
```

Comment:
I know this is just copypasted code, but could you take out the quotes. i.e. { 'originalGeoMean': ... } -> { orginalGeoMean: ... }